common/params_pyx.pyx

```diff
@@ -29,11 +29,13 @@ cdef extern from "common/params.h":
 
 
 def ensure_bytes(v):
-  return v.encode() if isinstance(v, str) else v;
+  return v.encode() if isinstance(v, str) else v
+
 
 class UnknownKeyName(Exception):
   pass
 
+
```

Comment:
is this a lint rule? if so, let's disable it and revert these changes

Reply:
Yep, it is one of the rules. I will revert the change, add new ignore and we are good to go.